src/main/resources/youtube/settings/xml/revanced_prefs.xml

```diff
@@ -30,6 +30,8 @@
         <Preference android:title="@string/revanced_alt_thumbnail_stills_about_title" android:selectable="false" android:summary="@string/revanced_alt_thumbnail_stills_about_summary" />
         <SwitchPreference android:title="@string/revanced_alt_thumbnail_stills_fast_title" android:key="revanced_alt_thumbnail_stills_fast" android:summaryOn="@string/revanced_alt_thumbnail_stills_fast_summary_on" android:summaryOff="@string/revanced_alt_thumbnail_stills_fast_summary_off" />
         <ListPreference android:entries="@array/revanced_alt_thumbnail_stills_time_entries" android:title="@string/revanced_alt_thumbnail_stills_time_title" android:key="revanced_alt_thumbnail_stills_time" android:entryValues="@array/revanced_alt_thumbnail_stills_time_entry_values" />
+        <SwitchPreference android:title="@string/revanced_alt_thumbnail_alternative_domain_enable_title" android:key="revanced_alt_thumbnail_alternative_domain_enable" android:summary="@string/revanced_alt_thumbnail_alternative_domain_enable_summary"/>
```

Comment:
1. "enable" is conventionally near the beginning of the string name. 

2. If the title string name says "enable", the summary string name should also say "enable"

3. "Enable" may be the incorrect the wrong word because it is usually only in the string name when the title starts with "Enable", such is by `revanced_enable_opus_codec_title` (where the title says "Enable OPUS codec")

Maybe "use" is a better word: 

Title: `revanced_alt_thumbnail_use_alternative_domain_title`
Summary: `revanced_alt_thumbnail_use_alternative_domain_summary`